test/model/workloads/WT-15311.workload

```diff
@@ -0,0 +1,25 @@
+config("database", "disaggregated=true")
+create_table(2, "table2", "Q", "Q")
```

Comment:
does Q here mean we only care about column store?

Reply:
no, right? q means integer.

Reply:
my bad, I most see `i` so got confused.